src/Symfony/Component/Ldap/Adapter/ExtLdap/Connection.php

```diff
@@ -165,7 +164,7 @@ private function connect()
 
     private function disconnect()
     {
-        if ($this->connection && \is_resource($this->connection)) {
+        if ($this->connection) {
```

Comment:
Are you sure about this one)

Reply:
That's the true bugfix here: `is_resource` checks won't work on 8.1. 
Theoretically it can't be anything else than `resource|LDAPConnection|false` so yes I am :)

Reply:
Should it be `if ($this->connection && $this->connection)`?

Reply:
Ok sorry nevermind...

Reply:
The change makes sense to me, but for that to work now means the additional check was irrelevant all the time ?

Reply:
Yes, this extra check was already not necessary before this PR. 
It's been originally introduced in https://github.com/symfony/symfony/pull/14602/files#diff-a9769aefb89bf09aa1c4cb1f73e97c499b0570421645e898efeecbbdbd7c1ae9R120, then moved to this class in https://github.com/symfony/symfony/pull/17560/files#diff-7f8586b60e3b3133d612459f8a613e07845e3b6cf57c212814ee6b2ff36445dfR83.